nucleus/osgi-platforms/felix/src/main/resources/config/osgi.properties

```diff
@@ -102,7 +102,7 @@ extra-system-packages=${jre-${java.specification.version}}, org.glassfish.embedd
 # Although Eclipselink imports these packages, in typical GlassFish installation,
 # Oracle JDBC driver may not be available as a bundle, so we ask user to install it in
 # java.ext.dirs and the bootdelegation helps there.
-eclipselink.bootdelegation=oracle.sql, oracle.sql.*
+eclipselink.bootdelegation=oracle.sql, oracle.sql.*,oracle.jdbc, oracle.jdbc.*
```

Comment:
I don't think this is needed now that we set org.osgi.framework.bundle.parent=app, EclipseLink and Oracle worked for the sample app even without it. But it causes no harm so we can leave it there.

Reply:
It was necessarry, eclipselink had trouble making connection at some different spot otherwise